src/general/Jukebox.cs

```diff
@@ -223,6 +248,14 @@ private void PlayTrack(AudioPlayer player, TrackList.Track track, string trackBu
 
         if (changedTrack || !player.Playing)
         {
+            var target = categories[PlayingCategory];
+            if (target.TrackTransition == MusicCategory.Transition.Crossfade)
```

Comment:
Could you add a TODO comment here that it might be nice to skip the fade if the *same* track is going to be played again? (so a special TrackTransition would be added that only triggers the fade if changedTrack is true) With microbe ambiance2 I think that would sound better when it loops that it doesn't have the fade in the middle of it.